tests/fixtures/fixtures_job.py

```diff
@@ -64,43 +73,43 @@ def job_mock(auth_mock, requests_mock):
 
 
 @pytest.fixture()
-def job_live(auth_live):
+def job_live(auth_live, project_id_live):
     job = Job(
         auth=auth_live,
-        project_id=auth_live.project_id,
+        project_id=project_id_live,
         job_id=os.getenv("TEST_UP42_JOB_ID"),
     )
     return job
 
 
 @pytest.fixture()
 def jobs_mock(auth_mock, requests_mock):
-    url_job_info = f"{auth_mock._endpoint()}/projects/{auth_mock.project_id}/jobs/{JOB_ID}"
+    url_job_info = f"{auth_mock._endpoint()}/projects/{PROJECT_ID}/jobs/{JOB_ID}"
     requests_mock.get(
         url=url_job_info,
         json={"data": {"xyz": 789, "mode": "DEFAULT"}, "error": {}},
     )
 
-    job1 = Job(auth=auth_mock, project_id=auth_mock.project_id, job_id=JOB_ID)
+    job1 = Job(auth=auth_mock, project_id=PROJECT_ID, job_id=JOB_ID)
 
-    url_job_info = f"{auth_mock._endpoint()}/projects/{auth_mock.project_id}/jobs/{JOB_ID_2}"
+    url_job_info = f"{auth_mock._endpoint()}/projects/{PROJECT_ID}/jobs/{JOB_ID_2}"
     requests_mock.get(url=url_job_info, json={"data": {"xyz": 789}, "error": {}})
 
-    job2 = Job(auth=auth_mock, project_id=auth_mock.project_id, job_id=JOB_ID_2)
+    job2 = Job(auth=auth_mock, project_id=PROJECT_ID, job_id=JOB_ID_2)
     return [job1, job2]
 
 
-@pytest.fixture()
-def jobs_live(auth_live):
+@pytest.fixture
```

Comment:
interesting!
I guess it makes no difference whether to instantiate or not.
We should remove it from other fixtures as well.

Reply:
Agree, I tried to reduce code scouting in this PR, we should have those detected when all pre-commit hooks are enabled